Dockerfile

```diff
@@ -0,0 +1,83 @@
+FROM harbor.k8s.libraries.psu.edu/library/ruby-2.6.6-node-15:20210507 as base
+ARG UID=2000
+
+USER root
+RUN apt-get update && \
+   apt-get install --no-install-recommends -y \
+   sqlite3 \
+   vim \
+   default-libmysqlclient-dev \
+   libsqlite3-dev \
+   shared-mime-info && \
+   rm -rf /var/lib/apt/lists*
+
+RUN useradd -u $UID app -d /app
+RUN mkdir /app/tmp
+RUN chown -R app /app
+USER app
+
+COPY Gemfile Gemfile.lock /app/
+RUN gem install bundler:2.1.4
+RUN bundle config set path 'vendor/bundle'
+RUN bundle install --deployment && \
+  rm -rf /app/.bundle/cache && \
+  rm -rf /app/vendor/bundle/ruby/*/cache
+
+
+COPY package.json yarn.lock /app/
+RUN yarn --frozen-lockfile && \
+  rm -rf /app/.cache && \
+  rm -rf /app/tmp
+
+
+CMD ["/app/bin/startup"]
+
+# Final Target
+FROM base as production
+
+# Clean up Bundle
+# RUN bundle config set path 'vendor/bundle'
+# RUN bundle install --without development test && \
+#   bundle clean && \
+#   rm -rf /app/.bundle/cache && \
+#   rm -rf /app/vendor/bundle/ruby/*/cache
+
+USER app
+
+COPY --chown=app . /app
+
+RUN RAILS_ENV=production \
+  SECRET_KEY_BASE=rails_bogus_key \
+  MYSQL_USER=mysql \
+  MYSQL_PASSWORD=mysql \
+  MYSQL_HOST=mysql \
+  MYSQL_DATABASE=mysql \
+  bundle exec rails assets:precompile && \
```

Comment:
I didn't think we were using asset pipeline. There is no `app/assets` directory. I thought everything was in webpacker.

Reply:
Nm. It's calling webpack.

Reply:
I was under the impression that assets:precompile was an abstraction to webpacker/sprockets/whatever else and the app just _did the right thing_ this does mirror what we do in prod today, but if we can streamline it i'd be for that, too 

https://github.com/psu-libraries/psulib_blacklight_deploy/blob/d3c8a434f8dcdcd112c425b749a56f3cdbc60050/roles/ansible-rails_app/handlers/main.yml#L10